Sources/MapboxMaps/Foundation/MapboxMap.swift

```diff
@@ -29,4 +30,91 @@ public final class MapboxMap {
+    ///   - bearing: The new bearing to be used by the camera.
+    ///   - pitch: The new pitch to be used by the camera.
+    /// - Returns: A `CameraOptions` that fits the provided constraints
+    public func camera(for coordinates: [CLLocationCoordinate2D],
+                       padding: UIEdgeInsets,
+                       bearing: Double?,
+                       pitch: Double?) -> CameraOptions {
+        return __map.cameraForCoordinates(
+            forCoordinates: coordinates.map(\.location),
+            padding: padding.toMBXEdgeInsetsValue(),
+            bearing: bearing.map(NSNumber.init),
+            pitch: pitch.map(NSNumber.init))
+    }
+
+    /// Calculates a `CameraOptions` to fit a list of coordinates into a sub-rect of the map.
+    ///
+    /// Adjusts the zoom of `camera` to fit `coordinates` into `rect`.
+    ///
+    /// - Parameters:
+    ///   - coordinates: The coordinates to frame within `rect`.
+    ///   - camera: The camera for which the zoom should be adjusted to fit `coordinates`. `camera.center` must be non-nil.
+    ///   - rect: The rectangle inside of the map that should be used to frame `coordinates`.
+    /// - Returns: A `CameraOptions` that fits the provided constraints, or `cameraOptions` if an error occurs.
+    public func camera(for coordinates: [CLLocationCoordinate2D],
+                       camera: CameraOptions,
+                       rect: CGRect) -> CameraOptions {
+        return __map.cameraForCoordinates(
+            forCoordinates: coordinates.map(\.location),
+            camera: camera,
+            box: ScreenBox(rect))
+    }
+
+    /// Calculates a `CameraOptions` to fit a geometry
+    ///
+    /// - Parameters:
+    ///   - geometry: The geoemtry that will be displayed within the viewport.
+    ///   - padding: The new padding to be used by the camera.
+    ///   - bearing: The new bearing to be used by the camera.
+    ///   - pitch: The new pitch to be used by the camera.
+    /// - Returns: A `CameraOptions` that fits the provided constraints
+    public func camera(for geometry: Geometry,
+                       padding: UIEdgeInsets,
+                       bearing: CGFloat?,
+                       pitch: CGFloat?) -> CameraOptions {
+        return __map.cameraForGeometry(
+            for: MBXGeometry(geometry: geometry),
+            padding: padding.toMBXEdgeInsetsValue(),
+            bearing: bearing.map(NSNumber.init),
+            pitch: pitch.map(NSNumber.init))
+    }
+
+    // MARK: - CameraOptions to CoordinateBounds
+
+    /// Returns the coordinate bounds corresponding to a given `CameraOptions`
+    ///
+    /// - Parameter camera: The camera for which the coordinate bounds will be returned.
+    /// - Returns: `CoordinateBounds` for the given `CameraOptions`
+    public func coordinateBounds(for camera: CameraOptions) -> CoordinateBounds {
+        return __map.coordinateBoundsForCamera(forCamera: camera)
+    }
```

Comment:
Not testing these for now (other than the 1 test which was migrated). They're passthroughs to core, so once we do a bit more work and can depend on an interface instead of an implementation, we could mock and ensure that the expected values are passed through.